client/src/components/CustomerDetails.js

```diff
@@ -0,0 +1,113 @@
+import React, { useState } from "react";
+import { gql, useMutation, useQuery } from '@apollo/client';
+
+import { useHistory } from "react-router-dom";
+
+const CUSTOMER_DETAILS = gql`
+    mutation EditCustomer($friendlyName:String! $phoneNumber:String!) {
```

Comment:
edit = "change existing users"
create = "create a new user"
make the names consistent

Reply:
I've solved this problem on the "delete customer" PR. You can check it there now.